prowler/providers/gcp/lib/service/service.py

```diff
@@ -8,11 +11,32 @@ def __init__(self, service, audit_info, region="global", api_version="v1"):
         self.service = service.lower() if not service.islower() else service
 
         self.api_version = api_version
-        self.project_ids = audit_info.project_ids
         self.default_project_id = audit_info.default_project_id
 
         self.region = region
         self.client = generate_client(service, api_version, audit_info)
+        self.project_ids = self.__is_api_active__(audit_info.project_ids)
```

Comment:
Include a comment here to say that the project_ids will be included in the API is active, or maybe we can change the name to `self.api_active_project_ids`

Reply:
Done